app/models/package_manager/pypi.rb

```diff
@@ -152,5 +152,14 @@ def self.project_find_names(project_name)
         project_name.gsub("_", "-"),
       ]
     end
+
+    # checks to see if the package exists on PyPI and the name matches the canonical name
+    def self.valid_project?(name)
```

Comment:
I think you could avoid this comment by naming the method `has_canonical_pypi_name?` or something along those lines

Reply:
Updated this method name in c808e9ecc4aa85a2d0666c094062561cf1f5e4f6